lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -63,21 +63,21 @@ <h3 class="error-msgs" id="error-msg"></h3>
          <h2>${_("Sales")}</h2>
          <div>
             %if section_data['total_amount'] is not None:
-              <span><strong>${_("Total CC Amount: ")}</strong></span><span>$${section_data['total_amount']}</span>
+              <span><strong>${_("Total Credit Card Purchases: ")}</strong></span><span>$${section_data['total_amount']}</span>
             %endif
             <span class="csv_tip">
               <div >
-              ${_("Click to generate a CSV file for all sales records in this course")}
-                <input type="button" class="add blue-button" name="list-sale-csv" value="${_("Download All Invoice Sales")}" data-endpoint="${ section_data['get_sale_records_url'] }" data-csv="true">
-                <input type="button" class="add blue-button" name="list-order-sale-csv" value="${_("Download All CC Sales")}" data-endpoint="${ section_data['get_sale_order_records_url'] }" data-csv="true">
+              ${_("Download a .csv file for all credit card purchases or for all invoices, regardless of status")}
+                <input type="button" class="add blue-button" name="list-sale-csv" value="${_("Download All Invoices")}" data-endpoint="${ section_data['get_sale_records_url'] }" data-csv="true">
+                <input type="button" class="add blue-button" name="list-order-sale-csv" value="${_("Download All Credit Card Purchases")}" data-endpoint="${ section_data['get_sale_order_records_url'] }" data-csv="true">
               </div>
             </span>
              <hr>
-             <p>${_("Enter the invoice number to invalidate or re-validate sale")}</p>
+             <p>${_("To cancel or resubmit an invoice, enter the invoice number below.")}</p>
             <span class="invalid_sale">
-               <input type="number" id="invoice_number" placeholder= "${_("Enter Invoice Number")}"/>
-               <input type="button" class="add blue-button" id="invalidate_invoice" value="${_("Invalidate Sale")}">
-               <input type="button" class="add blue-button" id="re_validate_invoice" value="${_("Revalidate Sale")}">
+               <input type="number" id="invoice_number" placeholder= "${_("Invoice Number")}"/>
+               <input type="button" class="add blue-button" id="invalidate_invoice" value="${_("Cancel Invoice")}">
```

Comment:
Yes, that was deferred work. It doesn't "do" anything - our PM never finished specifying the invalidation flows. I'd suggest removing it for now until we have real support for invoice invalidations (which should revoke any enrollments from the Redeem Codes that were produced from the Invoice). Obviously, there would need to be some UI messaging to the end user why his/her enrollment was revoked.